frontend/src/test/java/org/abs_models/frontend/typesystem/LocationTypeTests.java

```diff
@@ -318,38 +304,11 @@ public void testInferenceRetypeChecking() {
         SemanticConditionList e = m.typeCheck();
         m.typeCheck(new SemanticConditionList());
     }
-    
```

Comment:
Can you comment on why the test was removed?  (Inapplicable, fails with new implementation, ...)

Reply:
I was unsure what exactly was tested here. There are no assertions, it always failed due to the default type being set to INFER, which led to errors, and I believe it also failed for the old system.

Reply:
Ok, thanks.